### ISSUE TITLE
agent_groups misguided output when deleting group

### ISSUE THREAD

**Issue by havidarou** [wazuh/wazuh#2955]
|Wazuh version|Install type|Install method|Platform|
|---|---|---|---|
| 3.7.1 and 3.8.2 (that I know of) | Manager | Packages | Any |

Hello team,

I've noticed that, when removing a group from an agent that only belongs to it, we get the following output:

```
root@wazuh-manager:/# /var/ossec/bin/agent_groups -l -g default
2 agent(s) in group 'default':
  ID: 004  Name: DESKTOP-.
  ID: 005  Name: vagrant-centos-7-17.
root@wazuh-manager:/# /var/ossec/bin/agent_groups -r -i 004 -g default
Do you want to delete the group 'default' of agent '004'? [y/N]: y
Group 'default' unset for agent '004'.
root@wazuh-manager:/# /var/ossec/bin/agent_groups -l -g default
2 agent(s) in group 'default':
  ID: 004  Name: DESKTOP-.
  ID: 005  Name: vagrant-centos-7-17.
```

It looks like we can't remove a group from an agent that only belongs to one group, but `agent_groups` output is erroneous: `Group 'default' unset for agent '004'`.

Regards.

**Comment by DaveVG1** [comment]
It seems like it's fixed in 3.9 (https://github.com/wazuh/wazuh/commit/68462514f53421f3fc88d278f09b67403f249152):
```
root@vm-ubuntu4:/home/wazuh/src# agent_groups -l
Groups (2):
  default (1)
  grupo1 (0)
Unassigned agents: 0.
root@vm-ubuntu4:/home/wazuh/src# agent_groups -r -i 001 -g default
Do you want to delete the group 'default' of agent '001'? [y/N]: y
Error 1745: Agent only belongs to 'default' and it cannot be unset from this group.
```

"_It looks like we can't remove a group from an agent that only belongs to one group_". This isn't totally accurate, this is only true if the only group which the agent belongs to is `default`. In another case, the agent won't belong to the specific group anymore, but it will belong to `default`.

Regards.

**Comment by DaveVG1** [comment]
Hi @havidarou , as this seems to be fixed, let me close this issue.

If this causes any troubles in the future, don't hesitate to open it again.

Regards.
